---
title: "Role of wildlife in the spread and maintenance of antibiotic resistance (PRC2016020)"
author: 'SUMMARY'
header-includes: \usepackage[italian]{babel} \usepackage{setspace}\doublespacing \usepackage{float} 
                  \usepackage{booktabs} \usepackage{longtable} 
                  \usepackage{fontspec} \usepackage{caption}
                  \setmainfont{Times New Roman}
                  
              
output:
  pdf_document:
    latex_engine: xelatex
    #toc: yes
    number_sections: yes
    fig_caption: yes
    extra_dependencies: ["float"]
 
    
bibliography: amrRef.bib
link-citations: yes
linkcolor: blue
 

---

**Key words:** antibiotic-resistance, wildlife, resistome. 

 

**Introduction**

Interest in the role of wildlife in the spread and maintenance of antibiotic resistance has increased over the past 10 years. The main concern arises from the fear that any demonstration of a role of wildlife as a "reservoir" and "maintenance" of bacteria carrying resistance genes, contributing to the environmental contamination of resistant bacteria or genetic material of resistance, could compromise efforts implemented, especially in recent years, in the medical and veterinary fields to reduce the spread of the phenomenon of antibiotic resistance. Antibiotic resistance is in itself a natural phenomenon and as such it is not surprising that antibiotic-resistant strains are found in wild animals even in remote areas of the globe. Available data [@greig_scoping_2015] show that numerous species of wild animals are carriers of resistant antimicrobial bacteria in a wide range of habitats, which raises the question of their role in the dynamics of spread and maintenance at the interface between human populations, domestic animals and natural ecosystems. The presence in wildlife, normally not subjected to antibiotic treatments, of resistant bacterial strains and in general of resistance genes, is probably attributable to environmental contamination phenomena.
To define the role of wildlife in the complex mechanism of diffusion and maintenance of antibiotic resistance, it is necessary to acquire information on: which species of bacteria are most frequently found resistant to antimicrobial drugs in wild vertebrates and their phenotypic and above all genotypic characterization; how wild species are colonized by antibiotic-resistant bacteria and what exchanges of these bacteria take place between humans, domestic animals and wildlife; what characterizes the habitats most contaminated by antibiotic-resistant bacteria and finally which ecological traits favor colonization and potential infection with antibiotic-resistant bacteria in wildlife [@vittecoq_antimicrobial_2016]. Based on the available data on the spread in Italy and in the rest of the world of antibiotic resistance in wild fauna, an observational study was designed with the aim of collecting information on the presence and spread of antibiotic resistance in bacterial strains of the Enterobacteriaceae family. isolated from faeces of a wide spectrum of wildlife species, present in various provinces of Lombardy with different degrees of urbanization, thus proceeding to their phenotypic characterization of resistance and their genotypic characterization using metagenomic analysis methods.

**Materials and Methods**

In the period between the end of 2017 and the first months of 2019, a non-probabilistic sampling of convenience was carried out using both the samples collected during the surveillance activities of the regional wildlife plan of Lombardy (wild boars, wild ruminants, lagomorphs, birds ), and samples from surveillance activities of the Chronic Wasting Disease plan (deer and roe deer) and of the West Nile Disease plan (Cornacchie, Magpies). The stool samples were then processed using common classical microbiology methods to obtain the isolation of Enterobacteriaceae strains, identified by biochemical tests both in macro and micromethod. The isolated strains were tested with a panel of seven antibiotics (AMPICILLIN, TETRACYCLIN, CEFTIOFUR, COLISTIN, KANAMYCIN, GENTAMYCIN, ENROFLOXACIN) to define their phenotypic resistance profile. Strains of Escherichia coli found to be resistant to CEFTIOFUR were subjected to phenotypic and genotypic analyzes for the determination of the presence of ESBL enzymes. Ten pools of E. coli strains, grouped by species of origin and by geographical location, were analyzed by means of metagenomic techniques to define their genotypic resistance profiles. Phenotypic data were analyzed by Bayesian statistical methods to obtain estimates of the prevalence of carrier animals of resistant and multi-resistant strains and of the prevalence of resistant and multi-resistant strains and to study the relationship between territorial characteristics (anthropogenic gradient) and prevalence of carrier animals of resistant strains. The biodiversity of the different phenotypic profiles was studied through the definition of Renyi's profiles

**Results and discussion**
The sampling activity took place between September 2017 and December 2018 in seven provinces of Lombardy from a total of 223 municipalities. A total of 670 stool samples from 33 different wildlife species were collected. 911 bacterial strains identified as Enterobacteriaceae were isolated from the 671 stool samples analyzed. Escherichia coli (E.coli) represents 67.6% of the isolates, the remaining 32% of the strains is distributed on a total of 14 different genera. 910 antibiograms were performed. A total of 392 strains (43%) were Susceptible (S) to the panel of antibiotics tested; 423 (46%) Resistant (R) strains up to a maximum of 2 antibiotics and finally 95 (10%) strains were found to be Multidrug Resistant (MR), with resistance to 3 or more antibiotics. Altogether from 670 wildlife stool samples from which at least one _Enterobacteriaceae_ strain was isolated and on which the antibiogram was performed, at least one strain resistant to one or more antibiotics of the panel investigated was isolated in 421 samples, equal to an Overall Prevalence of 62 % (95% HPD: 58-67%). The variability between the species-groups is particularly wide: the highest prevalence is observed in CORVIDAE (84%), while in wild BOVIDAE the lowest (37%). Due to the low sample size, estimates with very large uncertainty are observed in the groups: HARE, OTHER BIRDS, WATER BIRDS, CARNIVORES and BIRDS OF PRAY. Out of 670 stool samples, 92 have at least one MULTI-RESISTANT strain (number of resistances> = 3), equal to a prevalence of 14% (95% HPD: 11% -16%). The variability between species groups is very wide: the prevalence of wild animals carrying multidrug-resistant strains varies from a minimum of 4% in the AQUATIC BIRDS group to a maximum of 88% in the HARE. The low number of samples of some species groups makes the estimates very uncertain. The association between the prevalence of resistant cases and the grazing area is conditionally dependent on the species group considered; this is a reasonable result from the biological point of view because it reflects the different frequentation of the different species in areas with different degrees of urbanization (the areas used for grazing tend to be present in areas with a lesser degree of urbanization). However, there is an increase in the probability of observing carrier animals of Enterobacteriaceae strains resistant to at least one antibiotic, in all species groups with different entities, except for the CORVIDAE group, in which, as the area used for grazing increases, it is observed a reduction in the risk of carrier animals. The data provided by this study suggest that, overall, the phenomenon of multi-resistance, at least as regards its phenotypic manifestation, is infrequent, equal to about 14% (95% HPD: 11% -16%) in the species and territories investigated. In the CORVIDAE group, most exposed to contamination from anthropomorphized environments it is equal to 21% (95% HPD: 15% -28%). Resistance against AMP and TET were the most frequent: out of 910 strains tested against AMP 388 are resistant (%), 342 strains are resistant to TET (). There is a very low frequency of resistance against other antibiotics: only 8% resistance against CFT, 5% against COL, 5% against ENR, 5% against KAN and only the 1.3% against GEN. Genomic identification of the 47 ceftiofur-resistant strains made it possible to identify Escherichia coli in 85% of cases and Escherichia fergusonii in 15%. In the strains of Escherichia coli, a prevalence of 10% (4/40) for the TEM gene and 2.50% (1/40) for SHV emerged. In Escherichia fergusonii strains, a prevalence of 14.29% (1/7) was found for the SHV gene. Overall, 39 different phenotypic profiles based on the observed resistances are observed. The most frequent profiles are: TET-AMP (141 strains), AMP (130 strains), TET (85 strains), CFT-TET-AMP (27 strains), COL-TET-AMP (14 strains), COL-TET ( 12 strains) and CFT-TET (12 strains), representing more than 80% of the strains. In the CERVIDAE population, a dominance profile is observed in terms of diversity with a lower diversity of co-resistance profiles with a strong dominance of a few profiles. On the contrary, the species groups BOVIDS, SUIDES, CARNIVORES, HARE, WATER BIRDS and OTHER BIRDS , show a profile of uniformity (evenness), therefore characterized by greater diversity without the presence of relatively dominant co-resistance profiles. The CORVIDAE and BIRDS OF PRAY populations show an intermediate diversity profile compared to the dominance of CERVIDAE and uniformity of the other species-groups. 1186 / 5000

The results of this study confirm the presence of resistant _Enterobacteriaceae_ strains and resistance genes in a wide spectrum of wildlife species of a vast territory of the Northen Italy with different territorial characteristics and anthropegenic pressure. The prevalence of multi-resistant strains is contained and this supports what has already been described in the literature [@hassell_clinically_2019-1], the hypothesis that the wildlife sector acts mainly as an accidental container of the environmental resistome in which the absence of continuous selective stimuli (administration of antibiotics) does not allow clonal expansion of resistant strains. From this point of view, the wildlife take on a role of sentinel of environments contaminated by the environmental resistance. The reduction of the environmental contamination of antibiotic residues and chemical substances represents the main safeguard to prevent resistant strains present in the microbiome of wild species from developing and expanding in a critical way, triggering contamination flows towards domestic animals and human populations, frustrating the efforts made act in recent years to reduce the phenomenon of antibiotic resistance.

**References**

